### DISCUSSION TITLE
Offline Desktop App

### DISCUSSION THREAD

**Discussion by j4s0n-c** [noisedeck/noisedeck-docs#22] [Questions and Issues] [answered]
Would the pro desktop app be able to be used without an internet connection? For live "VJ" use I think this is essential (in addition to MIDI input and Spout output). Most venues have slow / congested WIFI, if any. I do have a hotspot on my phone but even that won't work out in the desert...

**Comment by jesshewitt** [comment, marked as answer]
Yes, I think that is essential too. 👍

**Reply by j4s0n-c** [reply]
Excellent! Thank you for the awesome work so far! Excited for what's to come.

**Reply by j4s0n-c** [reply]
It seems everything (so far) works without internet as long as I am connected when I log in. Once the noise starts I can turn off WIFI and it just keeps going. Even rendering out frames. Sweet. Seems all you need is a way to verify a pro user offline and, good to go!
